README.rst

```diff
@@ -5,9 +5,9 @@ lib_i2s: I²S/TDM library
 ########################
 
 :vendor: XMOS
-:version: 6.0.0
+:version: 6.0.1
 :scope: General Use
-:description: I²S/TDM master and slave
+:description: I²S/TDM controller and target
```

Comment:
I have never heard the term target before! Whilst PC it may not be what a large number of engineers will understand.

Reply:
it's in the spec now so thought we should change https://www.nxp.com/docs/en/user-manual/UM11732.pdf